ni_measurement_plugin_sdk/discovery/_support.py

```diff
@@ -57,7 +57,12 @@ def _get_discovery_service_location() -> pathlib.PurePath:
 
 def _get_registration_json_file_path() -> pathlib.Path:
     if sys.platform == "win32":
-        return _get_nipath("NISHAREDDIR64") / "MeasurementLink" / "MeasurementLinkServices.json"
+        json_path = _get_nipath("NISHAREDDIR64") / "Services" / "Services.json"
+        if not json_path.exists():
```

Comment:
I don't see why we need this fallback. We plan to update the minimum required version of InstrumentStudio to 2024 Q3 in the docs, so I don't think we expect anyone to try our next release with older InstrumentStudio / MeasurementLink. @rfriedma ?

Reply:
presuming the developer has 2024 Q3+ installed then it would seem a fallback isn't required

Reply:
You are also renaming the package, so it's not like existing applications will automatically upgrade.

Reply:
I've removed the fallback with the latest update.